tox.ini

```diff
@@ -28,6 +28,12 @@ basepython = python
 deps = flake8
 commands = flake8 {posargs:.}
 
+[testenv:black]
+skip_install = true
+basepython = python
+deps = black
+commands = black --check {posargs:.}
```

Comment:
`--check` only checks the python files for incorrect formatting and will fail the test if unformatted files are found.